pages/common/lua.md

```diff
@@ -0,0 +1,15 @@
+# lua
+
+> A powerful, light-weight embedded programming language.
+
+- Start an interactive lua shell:
+
+`lua`
+
+- Execute a lua script:
+
+`lua {{script_name.lua}} {{--optional-argument}}`
+
+- Execute a string of lua:
+
+`lua -e '{{insert lua code here}}`
```

Comment:
I can see why he chose separate words, to emulate real code that would be passed. But we should either go the token way, e.g. `{{lua_code}}`, or use a concrete, hello-world-like example. What could the latter look like? It may be a good choice.